pkg/controller/watch/predicates.go

```diff
@@ -21,29 +21,16 @@ func CommonPredicates() predicate.Funcs {
 	}
 }
 
-// DeleteOnly returns a predicate that will filter out everything except the Delete event
-func DeleteOnly() predicate.Funcs {
-	return predicate.Funcs{
-		CreateFunc: func(ce event.CreateEvent) bool {
-			return false
-		},
-		UpdateFunc: func(ce event.UpdateEvent) bool {
-			return false
-		},
-		GenericFunc: func(ce event.GenericEvent) bool {
-			return false
-		},
-	}
-}
-
-func SelectNamespacesPredicate(namespaceMap map[string]bool) predicate.Funcs {
+func SelectNamespacesPredicate(namespaces []string) predicate.Funcs {
```

Comment:
please add a unit test for this function

Reply:
+1